queries.sql

```diff
@@ -44,3 +44,13 @@ SELECT AVG(weight_kg) AS average_weight from ANIMALS;
 SELECT MAX(escape_attempts) FROM animals GROUP BY neutered;
 SELECT MAX(weight_kg), MIN(weight_kg), species FROM animals GROUP BY species;
 SELECT AVG(escape_attempts), species FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* Day 3 */
+SELECT owners.full_name, owners.id, animals.name FROM animals INNER JOIN owners ON animals.owner_id = owners.id;
```

Comment:
- [ ] This query I believe should be for - **_What animals belong to Melody Pond?_**. If it is, then you are missing the where clause 👇🏾 

```sql
# What animals belong to Melody Pond?
SELECT * FROM animals
   LEFT JOIN owners
   ON animals.owner_id = owners.id
   WHERE full_name = 'Melody Pond';
```

Actually, it would have been easy if you used comments to label each query code with the question as I did above. The same with your screenshots in the Pull request. This will make it easy for a reviewer to know which query is for which.

Reply:
Remember to update your screenshots with the new code update. Well done 👏🏾